src/main/resources/hightouch/rules/hightouch_rules.json

```diff
@@ -0,0 +1,2030 @@
+[
+  {
+    "id": "ThirdParties.SDK.1o",
```

Comment:
Add Hightouch in rules, to not overlap with any actual rule passed from Privado rules
`ThirdParties.SDK.Hightouch.1o` like this